subprojects/code-quality/src/main/groovy/org/gradle/api/plugins/quality/FindBugsReports.java

```diff
@@ -39,7 +39,7 @@
      *
      * @return The findbugs HTML report
      */
-    SingleFileReport getHtml();
+    FindBugsHtmlReport getHtml();
```

Comment:
This was to be consistent with the `getXml()` method also on the `FindBugsReports`.

The type was switched back to `SingleFileReport` on the PR rework.